core/client-internal/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -42,85 +38,61 @@
   private static TachyonConf sTachyonConf;
   private static InetSocketAddress sMasterAddress;
   private static ClientMetrics sClientMetrics;
-  private static boolean sInitialized;
 
   static {
-    sInitialized = false;
     reset();
   }
 
   /**
-   * Initializes the client context singleton.
+   * Resets to the default Tachyon configuration and initializes the client context singleton.
    */
-  public static synchronized void reset() {
-    reset(new TachyonConf());
+  private static void reset() {
```

Comment:
`reset` is called from `ClientTestUtils`. `reset` is useful when you want to wipe the slate clean after making configuration changes. `init` is useful when you want your own configuration changes to be picked up by other pieces of context.